libs/events-import/api/jest.config.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+export default {
+  displayName: 'events-import-api',
+  preset: '../../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsconfig: '<rootDir>/tsconfig.spec.json'
+    }
+  },
+  testEnvironment: 'node',
+  transform: {
+    '^.+\\.[tj]s$': 'ts-jest'
+  },
+  moduleFileExtensions: ['ts', 'js', 'html'],
+  coverageDirectory: '../../../coverage/libs/events/import',
```

Comment:
if this is events/import, then the directory should be events/import of the file too